code/modules/jobs/jobs.dm

```diff
@@ -95,11 +95,11 @@ GLOBAL_PROTECT(exp_specialmap)
 		if(J.title == job_title)
 			return J.department_head //this is a list
 
-/proc/get_full_job_name(job)
+/proc/get_full_job_name(job) // this only gets used once in the entire code base lmao, and it's in some vocal cord surgery

```

Comment:
use ftl comment or remove this so i know its from us :P

Reply:
fine :P